src/translate-adapter.ts

```diff
@@ -66,8 +66,14 @@ parser
 			description:
 				"Path to the english i18n file, multiple files are possible",
 		},
+		languages: {
+			type: "string",
```

Comment:
What happens if we add `choices: ...,` here using all possible values for `ioBroker.Languages`? If we are lucky, the signature of `parseOptions` below could actually match.